src/components/global/Input.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+function Input({ errorMessage, label, value, onChange }) {
+  return (
+    <div className="input-container text-black">
+      <input
+        type="text"
+        className="input-outlined input-outlined-cyan"
+        autocomplete="off"
+        placeholder=" "
+        value={value}
+        onChange={onChange}
+      />
+      <label
+        style={{ backgroundColor: "white" }}
+        for="email"
+        className="input-label"
```

Comment:
Suggestion : Use variables for color , dont use direct colors , it will be difficult for you to scale the project by hard coding values. Keep that in mind